### ISSUE TITLE
`roles-add` command throws error

### ISSUE THREAD

**Issue by AceAttorneyMaster111** [gavwin/toasty#4]
When trying to add a role to the roles-list in a guild, Toasty raises an unexpected `TypeError`.
Error text: `TypeError: Cannot read property 'includes' of undefined`.

I looked at the file containing the command in question (/src/commands/config/roles-add.js), and the only instance of "includes" is here:
```javascript
if (data[msg.guild.id].roles.includes(role.name)) return msg.reply(`:no_entry_sign: The role, **${role.name}** is already in the server roles list.`);
```

`data` refers to the array containing server data evaluated from JSON. As a newcomer to this repository, I'm not sure where things are defined, but somehow `.roles` is returning `undefined`. I will continue investigating, but ideas on where to look would be greatly appreciated.

Thanks for making such a cool bot!

**Comment by AceAttorneyMaster111** [comment]
I believe I have figured out the problem.

Code snippet for reference:
```javascript
        if (!data[msg.guild.id]) data[msg.guild.id] = { roles: [] };
        if (data[msg.guild.id].roles.includes(role.name)) return msg.reply(`:no_entry_sign: The role, **${role.name}** is already in the server roles list.`);
        data[msg.guild.id].roles.push(role.name);
```
The code accounts for the situation in which there is no data at all by creating the data entry. However, it has no defined behavior if there is data, but no roles data.

If this is the problem, I have fixed it on my fork, but am not sure how to test it since I don't have permission to host my fork on a test server. How does one test changes to the code except by running them on the live bot?
